### ISSUE TITLE
check config can cause eternal reboot

### ISSUE THREAD

**Issue by bpwats** [home-assistant/hassio-addons#1577]
## The problem

The media folder is new in 115 beta. I had it filled up with nearly 10 GB of mp3 files, on a 32 GB SD card, besides a handful of backups. When check-config is run, the contents of the media folder is also copied. This caused the SD-card to run out of space. Only 3GB free space remaining before check-config started. One cannot access and control HA anymore. Pulling the cord on the RaspBerry Pi 4 does not help, it will eternally try to reboot. Eventually I put the SD-card in a Linux environment and was able to forcefully delete the MP3-folders (and some older backups). This allowed the RPi to boot again. 

## Environment

- Add-on with the issue:
Check Home Assistant configuration
- Add-on release with the issue: 
3.5.0
- Last working add-on release (if known): 
- Operating environment (OS/Supervised):
HassOS 4.12, Home Assistant 0.115.0b9

## Problem-relevant configuration

```yaml
default_config:
```

## Traceback/Error logs

```txt
check-config log stops half-way the copy function of the files in the media folder. HA becomes unresponsive.
```

## Additional information
Would be a good idea to exclude the media folder from copying? Only config is needed to compare?

**Comment by MartinHjelmare** [comment]
Thanks for the report, and sorry about the trouble. We'll move the default media folder out of the config folder before 0.115 is released. That should also fix this issue.

**Comment by balloob** [comment]
Fixed in #40071
